CHCSVParser/CHCSVParser/CHCSVParserOptions.swift

```diff
@@ -1,9 +0,0 @@
-//
-//  CHCSVParserOptions.swift
-//  CHCSVParser
-//
-//  Created by Dave DeLong on 3/7/15.
-//
-//
-
-import Foundation
```

Comment:
this file was empty, and SPM was complaining having mixed Obj-c and Swift sources so I just removed it